README.md

```diff
@@ -110,6 +110,20 @@ def test_view_function_user_with_purchases(pg):
    from pytest_mock_resources import create_mysql_fixture
    ```
 
+## Async
+It's also possible to create and use async engines (postgres only, sqlalchemy >= 1.4, python >= 3.6):
+```python
+import pytest
+from sqlalchemy import text
+from pytest_mock_resources import create_postgres_fixture
+
+postgres_async = create_postgres_fixture(async_=True)
+
+@pytest.mark.asyncio  # pytest-asyncio needs to be installed
```

Comment:
rather than this comment, should `pytest-asyncio` be an optional main requirement?

Reply:
Hmmm. I'm kinda undecided on this. It would be nice if the was installed alongside the other async dependencies. However, there is always a high possibility that the user has already installed it since it's a package that one tends to use when testing async code.

Reply:
in one of my below comments, i actually backtracked. i think we can avoid even needing a handle on a loop, so there'd be no real **need** to depend on it anyways.